sormas-base/setup/import-to-truststore.sh

```diff
@@ -0,0 +1,104 @@
+fi
+if [[ -z "${SORMAS2SORMAS_DIR}" ]] || [[ ! -d "${SORMAS2SORMAS_DIR}" ]]; then
+  DEFAULT_SORMAS2SORMAS_DIR="${ROOT_PREFIX}/opt/sormas2sormas"
+  if [[ -d "${DEFAULT_SORMAS2SORMAS_DIR}" ]]; then
+    SORMAS2SORMAS_DIR="${DEFAULT_SORMAS2SORMAS_DIR}"
+  else
+    while [[ ! -d "${SORMAS2SORMAS_DIR}" ]]; do
+		  read -r -p "Please specify a valid sormas2sormas directory: " SORMAS2SORMAS_DIR
+	  done
+	  export SORMAS2SORMAS_DIR
+  fi
+fi
+TRUSTSTORE_FILE_NAME=sormas2sormas.truststore.p12
+
+TRUSTSTORE_FILE=${SORMAS2SORMAS_DIR}/${TRUSTSTORE_FILE_NAME}
+NEW_TRUSTSTORE=false
+if [ ! -f "${TRUSTSTORE_FILE}" ]; then
+  NEW_TRUSTSTORE=true
+  echo "${TRUSTSTORE_FILE_NAME} not found. A new truststore file will be created."
+fi
+
+while [[ -z "${SORMAS_S2S_TRUSTSTORE_PASS}" ]]; do
+  read -sp "Please provide the password for the truststore: " SORMAS_S2S_TRUSTSTORE_PASS
+  echo
+done
+
+read -p "Please provide the file name of the certificate to import. It should be located inside the /sormas2sormas folder: " CRT_FILE_NAME
+CRT_FILE=${SORMAS2SORMAS_DIR}/${CRT_FILE_NAME}
+while [[ -z "${CRT_FILE_NAME}" ]] || [ ! -f "${CRT_FILE}" ]; do
+  echo "File not found in /sormas2sormas folder."
+  read -p "Please provide the file name of the certificate to import. It should be located inside the /sormas2sormas folder: " CRT_FILE_NAME
+done
+
+# import crt
+echo "Importing certificate into truststore..."
+if [[ ${NEW_TRUSTSTORE} = true ]]; then
+  openssl pkcs12 -export -nokeys -out "${TRUSTSTORE_FILE}" -password pass:"${SORMAS_S2S_TRUSTSTORE_PASS}" -in "${CRT_FILE}"
+  #update properties
+  if [[ -z ${SORMAS_PROPERTIES} ]]; then
+	  echo "sormas.properties file was not found."
+    echo "Please add the following properties to the sormas.properties file:"
+    echo "sormas2sormas.truststoreName=${TRUSTSTORE_FILE_NAME}"
+    echo "sormas2sormas.truststorePass=${SORMAS_S2S_TRUSTSTORE_PASS}"
+  else
+    # remove existing properties and empty spaces at end of file
+    sed -i "/^# SORMAS to SORMAS truststore data/d" "${SORMAS_PROPERTIES}"
+    sed -i "/^sormas2sormas\.truststoreName/d" "${SORMAS_PROPERTIES}"
+    sed -i "/^sormas2sormas\.truststorePass/d" "${SORMAS_PROPERTIES}"
+    sed -i -e :a -e '/^\n*$/{$d;N;};/\n$/ba' "${SORMAS_PROPERTIES}"
+    # add new properties
+    {
+      echo;
+      echo "# SORMAS to SORMAS truststore data";
+      echo "sormas2sormas.truststoreName=${TRUSTSTORE_FILE_NAME}";
+      echo "sormas2sormas.truststorePass=${SORMAS_S2S_TRUSTSTORE_PASS}";
+    } >> "${SORMAS_PROPERTIES}"
+fi
+else
+  # export existing certificates to temporary file
+  TEMP_FILE=${SORMAS2SORMAS_DIR}/tempcert.pem
+  openssl pkcs12 -in "${TRUSTSTORE_FILE}" -password pass:"${SORMAS_S2S_TRUSTSTORE_PASS}" -out ${TEMP_FILE}
+
+  # create new truststore with the new certificate and the certificates from the temporary file
+  openssl pkcs12 -export -nokeys -out "${TRUSTSTORE_FILE}" -password pass:"${SORMAS_S2S_TRUSTSTORE_PASS}" -in "${CRT_FILE}" -certfile ${TEMP_FILE}
```

Comment:
It seems that the java Keystore ignores plain certificates without alias/friendlyName.
With openssl there seems to be no way to add several certificates with friendlyNames.
So we probably should require Java to be installed and use keytool instead.